varats-core/varats/utils/util.py

```diff
@@ -0,0 +1,22 @@
+"""Small helper methods that implement common functionalities."""
+
+import typing as tp
+
+FunctionType = tp.TypeVar("FunctionType")
+
+
+def static_vars(**kwargs) -> tp.Any:
+    """
+    Decorates a function with static variables, passed with kwargs.
+
+    @static_vars(var_name=value)
+    def func(...):
```

Comment:
Make this an actual code block.

The following should work but is not tested:

```suggestion
    For example::

        @static_vars(var_name=value)
        def func(...):
```

If the highlighting is wrong, an additional `.. code-block:: python` should help.